app/src/main/java/org/mozilla/fenix/library/history/viewholders/HistoryListItemViewHolder.kt

```diff
@@ -43,12 +48,13 @@ class HistoryListItemViewHolder(
         }
     }
 
+    @Suppress("UndocumentedPublicFunction")
```

Comment:
Can you add a comment for this? We shouldn't be suppressing `UndocumentedPublicFunction` since this is easily avoidable.

Reply:
Do you mean removing the unused default value?

Reply:
I am assuming removing the unused default value triggered the lint error here? You can either undo it or keep the change but document the public function. We don't currently `@Suppress("UndocumentedPublicFunction")` and I am looking to keep it that way.

Reply:
Yes, it does trigger the lint error. May be if I rebase to the initial commit and revoke that bind method to the original form it might be okay.

A bind method cannot really have a meaningful documentation, can it?